content/help/addons/hero-image.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Hero Image
+tags:
+- Addons
+- Hero Image
+- Image upload
+category: help
+menu:
+  help:
+    parent: addons
+    identifier: addon-hero-image
+versioning:
+  added: 2.6
+---
+
+## Overview
+
+{{% cloudfeature %}}
+
+This plugin provides an easy way to have a custom background image for every page of the forum. It allows you to upload this image in the dashboard and access it through a [Smarty Template](/developer/smarty/functions/#function-hero-image-url) or [Customize Theme](/help/appearance/custom-theme).
+
+It also allows you override an the main image on a per-category basis. Overriding this background image for a category will also override the background image for every category or discussion inside of that category. This allows you to have a tree of background images through your forum, managed through the dashboard.
+
+<aside class="note">In order to be flexible this plugin does __not__ provide an out of the box solution for existing themes. Integrating the image will require custom theming work to be done. It's best to be knowledgeable in both HTML and CSS before using this plugin.</aside>
```

Comment:
Good note! Add links to custom theming in Vanilla /help/appearance/custom-theme/ and http://127.0.0.1:1313/developer/addons/theme-quickstart/